webapp/pages/terms-and-conditions.vue

```diff
@@ -1,14 +1,72 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAc') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t('site.termsAc') }} ( vom: {{ latestDate_TaC }})</ds-heading>
     </ds-space>
+    <div v-if="confirmed === true">
+      <ds-section class="hasConfirm" primary>
+        <ds-flex>
+          <ds-flex-item>
+            <ds-logo inverse></ds-logo>
+          </ds-flex-item>
+          <ds-flex-item width="4">
+            <ds-heading>you have confirmed</ds-heading>
+            <ds-text>
```

Comment:
```suggestion
            <ds-text v-if="confirmed">
```

**This** is where I would jump in with an if-clause. Because you have to replace the whole text.